integration-tests/gradle/projects/it-multiplatform-0/gradle.properties

```diff
@@ -1 +1,3 @@
 dokka_it_kotlin_version=1.7.10
+kotlin.mpp.enableGranularSourceSetsMetadata=true
+kotlin.native.enableDependencyPropagation=false
```

Comment:
What happens if you don't specify these two flags, will it not work?

I don't think our average user is going to have that enabled

Reply:
These flags are enabled by default since Kotlin 1.6.20. See [HMPP](https://kotlinlang.org/docs/multiplatform-hierarchy.html)
It will not work for earlier Kotlin versions.

Reply:
Oh, sweet. 

A comment would not hurt, imo. Otherwise future maintainers might be scared to remove it. Something along the lines of
```
// these flags are enabled by default since 1.6.20.
// remove when this test is executed with Kotlin >= 1.6.20
```